risc0/zkp/src/lib.rs

```diff
@@ -17,6 +17,7 @@
 #![deny(rustdoc::broken_intra_doc_links)]
 
 extern crate alloc;
+extern crate ff;
```

Comment:
This shouldn't be needed

Reply:
If I don't put it there I get a ton of errors.